docs/style-guide.rst

```diff
@@ -560,7 +560,7 @@ Yes::
         return x + 1;
     }
 
-    function increment(uint x) public pure onlyowner returns (uint) {
+    function increment(uint x) public pure onlyOwner returns (uint) {
```

Comment:
Thanks for the correction! I see that there are a few more instances of lowercase `onlyowner` in this file (and not they're not used to illustrate the style that's not recommended). Could you fix them too?

Reply:
Hi @cameel, ✅ Indeed, I added commit a90bf95fcb081f999019b4a30b96968962710a48 with the remaining typos fixed.